projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/quote/quote.e2e-flaky.cy.ts

```diff
@@ -25,44 +35,186 @@ context('Quote', () => {
       },
     };
     cy.cxConfig(globalMessageSettings);
+    quote.log('visit default url', 'beforeEach');
     cy.visit('/');
-    quote.login(EMAIL, PASSWORD, USER);
+    quote.log(`login as ${BUYER_USER}`, 'beforeEach');
+    quote.login(BUYER_EMAIL, BUYER_PASSWORD, BUYER_USER);
+    quote.log('registerGetQuoteRoute', 'beforeEach');
```

Comment:
```suggestion
    quote.log('registerGetQuoteRoute', 'beforeEach');
```
delete it - the registered routes are printed out by cypress anyways-

Reply:
done